tests/evaluation/aggregates_complex/aggregates_complex.dl

```diff
@@ -0,0 +1,5 @@
+// Souffle - A Datalog Compiler
```

Comment:
The copyright still isn't quite right, but I see now that this test is also blank. Remove this blank test case and add the working test in a later PR.